### ISSUE TITLE
About the few-shot experiments

### ISSUE THREAD

**Issue by aFewThings** [ilyassmoummad/ProtoCLR#1]
Hi, thank you for sharing your work. I have several questions on your study. 

1. In the reported experiments, did the reference systems train on support set using K audios and labels in soundscape datasets?

For instance, have you fine-tune all layers of Perch model on 1 or 5-shot support examples, and test on query set?

2. Classes of focal recording dataset XC cover all classes in soundscape datasets?

**Comment by ilyassmoummad** [comment]
Hi,

Thank you for your interest in our research!

1. Yes, the support set was sampled directly from the soundscape datasets, and the remaining samples from each dataset were used as the query set. In this study, we utilized pre-trained models without any fine-tuning. Exploring the adaptation of these models with K samples to handle domain shifts (from focal recordings to soundscapes) is an interesting and valuable research direction.

2. Yes, the classes of the focal recording dataset (XC) cover the classes present in the soundscape datasets. We use the same datasets as the benchmark BIRB: https://arxiv.org/abs/2312.07439

Please feel free to reach out if you have more questions or need further clarification on any aspects of our study.

**Comment by aFewThings** [comment]
Thank you for your fast response. I think this is an important research topic, and I hope you have good results for further publication.

**Comment by aFewThings** [comment]
Hi, I have an another question to follow your experiment.

Are the pre-trained reference systems evaluated by only taking the encoder except the classification layer and measuring the distance between the query sample and the prototypes on the support set?

**Comment by ilyassmoummad** [comment]
Hello! 

Exactly, we remove the classifier for models trained with cross-entropy, or the projector for models trained with a contrastive loss, and only use the encoder to extract the features, that are then used to compute the distances between query samples and the prototypes in the support set.

**Comment by aFewThings** [comment]
It's interesting approach, thank you!
